database/createDb.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE users (id integer PRIMARY KEY, username STRING NOT NULL);
+CREATE TABLE files (id integer PRIMARY KEY, filename STRING NOT NULL, data BLOB, owner integer, FOREIGN KEY(owner) REFERENCES users(id));
```

Comment:
Comments about `ROW_ID` and its relevance to the two `id` columns would be great!

Reply:
Can you give an example of what you mean for these?

Reply:
Maybe something like this:
```suggestion
-- 'id' column is an alias for SQLite's ROW_ID column, which is automatically populated
CREATE TABLE files (id integer PRIMARY KEY, filename STRING NOT NULL, data BLOB, owner integer, FOREIGN KEY(owner) REFERENCES users(id));
```